__tests__/endpoints.test.js

```diff
@@ -127,4 +127,54 @@ describe("/api/articles", () => {
         })
     })
     
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: returns with and array of comments with correct keys for article with specific ID", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const comments = body[0]
```

Comment:
body should be an object with a key of comments, it seems like body is an array here which is not which we should be expecting.